server/app.js

```diff
@@ -137,6 +134,12 @@ module.exports = function createApp(
 
   const healthcheck = healthcheckFactory(config.nomis.authUrl)
 
+  // Add services to server
+
+  const DataServices = { calendarService }
+
+  app.set('DataServices', DataServices)
```

Comment:
Is your preference to do this inline eg:

```
  app.set('DataServices', { calendarService })
```

I'd split it out as the line might get a big long with all the services in it, not I don't have a strong preference on this.

Also curious why naming the const the same is convoluted for you? 🤔 I thought it made it easier to read as we are naming the thing, then adding the thing to app under its name

Reply:
So, my perspective is that I'm doing a set, which is equivalent to a declaration anyway 🤔, so I'm happy with the raw var in there unless it gets to unwieldy, but I don't think it will? Even if it has to go multiline. To my mind, the naming the const was just an extension of linting rules not liking multi arguments in the same code with the same name, even if they have differing scope.
```
app.set('DataServices', { 
  calendarService,
  service2,
  service3,
})
```
Ultimately its a minor comment thats already gone beyond my original intention though now :-D
It sits outside our coding standards so if you prefer the double declaration then I'll merge 👍

Reply:
Yeah, that looks way more readable than it did in my head 😄